Sources/Sentry/include/HybridPublic/SentryBinaryImageInfo.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryBinaryImageInfo : NSObject
```

Comment:
`m`: Now that we simply pass the required properties as parameters to the LoadValidator, do we still need this extra class? If not, we could undo these changes and keep it as is.